frameworks/MLPlanSKLearn/requirements.txt

```diff
@@ -0,0 +1,7 @@
+liac-arff==2.4
+numpy>=1.15,<2.0
+scipy>=1.5,<1.6
+pandas>=0.23,<1.0
+tpot>=0.11.0,<0.12
+xgboost>=1.1.0,<1.2
+scikit-learn>=0.22.2
```

Comment:
How can I then reorder the dataset (as it is also done for AutoWEKA) such that the target is in the last column?

Reply:
> How can I then reorder the dataset (as it is also done for AutoWEKA) such that the target is in the last column?

Oh I see, we've got a problem here.
When running in a venv, your code can use any function from `amlb.utils` module because this one has minimmal dependencies (only yaml), but by it can't use `reorder_dataset` because this function is in `amlb.datautils`, this last module depending on `pandas`, `sklearn` and so on, so I didn't want to have all those dependencies by default in all venvs of course. 
It's possible to do the reordering in the `__init__.py` though, before calling the subprocess.
Let me try a couple a things, I'll come back to you with a solution (I hope).

Reply:
That'd be great!